docs/rules/no-debug.md

```diff
@@ -28,12 +28,6 @@ const { screen } = require('@testing-library/react');
 screen.debug();
 ```
 
-If you use [custom render functions](https://testing-library.com/docs/example-react-redux) then you can set a config option in your `.eslintrc` to look for these.
```

Comment:
This LGTM, I just wonder if we should mention that it can be configured via settings?

Reply:
Yes, I'm leaving all the documentation stuff for the last step after I've migrated all the rules. I'll mention the custom render settings in the README, but not sure if we should mention it on each rule depending on `render` (which is what I understand you are suggesting).

Reply:
Yea, correct. While I was writing it I was also considering if it's worth it to mention in for every rule or not 😅.
I think having a global doc is good.